[2023-08-30 19:23:54,492] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: entregable3.extract_and_transform_data manual__2023-08-30T19:23:52.788851+00:00 [queued]>
[2023-08-30 19:23:54,507] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: entregable3.extract_and_transform_data manual__2023-08-30T19:23:52.788851+00:00 [queued]>
[2023-08-30 19:23:54,508] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-08-30 19:23:54,511] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2023-08-30 19:23:54,512] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-08-30 19:23:54,532] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): extract_and_transform_data> on 2023-08-30 19:23:52.788851+00:00
[2023-08-30 19:23:54,537] {standard_task_runner.py:52} INFO - Started process 604 to run task
[2023-08-30 19:23:54,541] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'entregable3', 'extract_and_transform_data', 'manual__2023-08-30T19:23:52.788851+00:00', '--job-id', '101', '--raw', '--subdir', 'DAGS_FOLDER/entregable3.py', '--cfg-path', '/tmp/tmppvtq2ter', '--error-file', '/tmp/tmppusyao5k']
[2023-08-30 19:23:54,543] {standard_task_runner.py:80} INFO - Job 101: Subtask extract_and_transform_data
[2023-08-30 19:23:54,636] {task_command.py:371} INFO - Running <TaskInstance: entregable3.extract_and_transform_data manual__2023-08-30T19:23:52.788851+00:00 [running]> on host eebeded14cb4
[2023-08-30 19:23:54,723] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Yamil
AIRFLOW_CTX_DAG_ID=entregable3
AIRFLOW_CTX_TASK_ID=extract_and_transform_data
AIRFLOW_CTX_EXECUTION_DATE=2023-08-30T19:23:52.788851+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-08-30T19:23:52.788851+00:00
[2023-08-30 19:24:02,040] {logging_mixin.py:115} INFO - No hay nuevos datos para cargar en la tabla.
[2023-08-30 19:24:02,043] {python.py:173} INFO - Done. Returned value was: None
[2023-08-30 19:24:02,086] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=entregable3, task_id=extract_and_transform_data, execution_date=20230830T192352, start_date=20230830T192354, end_date=20230830T192402
[2023-08-30 19:24:02,150] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-08-30 19:24:02,178] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
